metrics/vocabulary/vocabulary.go

```
@@ -70,12 +70,14 @@ func WriteCSV(v *metrics.Vocabulary, filename string) {
 func WritePb(v *metrics.Vocabulary) {
 	bytes, err := proto.Marshal(v)
```

Please make this function return err (instead of printing errors and exiting)

---

thanks!